### ISSUE TITLE
Annotating human gene to viral processes

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#1507]
Is it correct to annotation Humam/Mouse  SSB to
	IRES-dependent viral translational initiation?

This isn't it's role in human, and results in it becoming annotated to "cytoplasmic" translation, when it is not involved in this process?
(plus lots of propagation).

This doesn't seem to be a good way to annotate viral processes?

**Comment by vanaukenk** [comment]
Hi @ValWood 
It looks like there are a number of IDA annotations to GO:0075522 'IRES-dependent viral translational initiation' for human proteins, so it would be good for the annotation group to review how we want to annotate the role of host proteins during viral infection.  I took a quick look at our documentation and don't see specific guidelines for host proteins, but may be missing it.
I forgot - do you have a Protein2GO account where you can flag these as annotation disputes?  If not, we could forward this thread to Sylvain for comment, as I think some of these annotations originally came from SIB.

**Comment by deustp01** [comment]
I think I remember from a lecture to students by a local virologist / gene expression expert, an assertion that the same mammalian IRES proteins co-opted by some viruses are also used by the mammal for responses to stresses like heat shock.

**Comment by ValWood** [comment]
I remember thinking that when the way we did this changed from the pathogen branch to use the same terms that it could be a problem. I don't think I really realised what was happening fully until today.....

I have spotted a number of instances  where the virus co-opts a host process, and the host gene is annotated to the  viral usage, which is a process which exists in the host but the viral use may not match the host process . This could really cause problems with GO  for analysing human data if it is heavily annotated to viral process usage......

I can't think of any way to prevent this other than to put back in a separate branch of GO.

**Comment by pgaudet** [comment]
It's very possible that viruses use the host's transcription factors,
however it is not the role of the human protein to transcribe viral
proteins, so I would say that these annotations are outside the score of
GO.

- Pascale

**Comment by ValWood** [comment]
Also human PCBP2. It is some type of RNA binding protein,

but it becomes annotated to
	IRES-dependent viral translational initiation
        and
viral RNA genome replication

The RNA genome replication , although odd does not harm but IRES-dependent viral translational initiation will give a "translation" annotation.

**Comment by dianeoinglis** [comment]
I agree with the above discussion about host proteins and transcriptional/translational processes involving viruses. A word of caution. Consider that metazoa have evolved to deal with viruses since pre-neandrethal times, some host proteins most certainly evolved for antiviral purposes. Cytokines such as the Interferons are an example. Whatever decision is made, I hope all host proteins are not excluded for this reason.

**Comment by ValWood** [comment]
an e.g. https://github.com/geneontology/go-annotation/issues/1715

**Comment by suzialeksander** [comment]
I'm assuming this is stale, if there are current instances of any questionable annotations please open a new ticket

**Comment by ValWood** [comment]
This is still very current. See recent paper post on slack.
